lib/webpacker/env.rb

```diff
@@ -0,0 +1,28 @@
+# Singleton registry for setting NODE_ENV based on config/webpack/paths.yml
```

Comment:
maybe we need to change `setting NODE_ENV` to `determining NODE_ENV`, as it no longer set's NODE_ENV, just calculates it's value for communicating when communicating with webpacker